app/views/shared/_place_form.html.erb

```diff
@@ -0,0 +1,69 @@
+<div class="col-12 col-sm-9 col-md-6 col-lg-4">
```

Comment:
đổi tên file => _places.html.erb

Reply:
lưu ý file nào dùng chung cho toàn dự án thì mới cho vào shared

Reply:
cái form này dùng toàn cho hai trang home vs trang profile thì có nên dùng k a

Reply:
có tách ra là đúng em nhé nhưng tên file chưa chuẩn lắm